src/components/content/catalog/services/details/ServiceProvider.tsx

```diff
@@ -49,6 +49,9 @@ function ServiceProvider({
     isViewDisabled: boolean;
     setIsViewDisabled: (isViewDisabled: boolean) => void;
 }): React.JSX.Element {
+    const [serviceRegisterStatus, setServiceRegisterStatus] = useState<serviceRegistrationState>(
```

Comment:
not sure if we need the state to be stored here. The state is in the backend. Means the state is available in the response object and we must reuse it.